package.json

```diff
@@ -33,6 +33,7 @@
 	],
 	"devDependencies": {
 		"@sindresorhus/tsconfig": "~0.7.0",
+		"@types/eslint": "^8.4.1",
```

Comment:
For this to work, `@types/eslint` would have to be a dependency rather than a dev dependency.

Though I don't like the idea of this module starting to rely on lots of other type modules. I think it should be self-contained.

We don't know if the version of `eslint` used in the project is `8.x`, `7.x` or something else. We don't even know if they use `eslint` at all.